joern-cli/frontends/php2cpg/src/main/scala/io/joern/php2cpg/astcreation/AstForTypesCreator.scala

```diff
@@ -66,7 +116,13 @@ trait AstForTypesCreator(implicit withSchemaValidation: ValidationMode) { this:
         typeDeclMember.astParentType(NodeTypes.TYPE_DECL)
       }
 
+      val metaTypeDeclMember = NewMember()
+        .name(s"${className}.<class>")
+        .code(s"${className}.<class>")
+        .dynamicTypeHintFullName(Seq(s"${classFullName}.<class>"))
+
```

Comment:
There seem to be no tests for this. Also make sure to that this is tested in the CS data flow tests.